packages/express/src/types.ts

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import { createClerkClient } from '@clerk/backend';
+import type { AuthenticateRequestOptions, AuthObject } from '@clerk/backend/internal';
+import type { Request as ExpressRequest, RequestHandler } from 'express';
+
+export type ExpressRequestWithAuth = ExpressRequest & { auth: AuthObject };
+
+export type ClerkMiddlewareOptions = AuthenticateRequestOptions & { debug?: boolean };
+
+/**
+ * Middleware for Express that handles authentication and authorization with Clerk.
+ * For more details, please refer to the docs: https://beta.clerk.com/docs/backend-requests/handling/express
```

Comment:
In other places we link to `https://beta.clerk.com/docs/references/express/overview?utm_source=github&utm_medium=express`, should we also use this here?